src/test/scala/za/co/absa/hyperdrive/trigger/scheduler/executors/spark/SparkEmrClusterServiceTest.scala

```diff
@@ -56,7 +56,7 @@ class SparkEmrClusterServiceTest
     with BeforeAndAfter {
   implicit override def executionContext: ExecutionContext = scala.concurrent.ExecutionContext.Implicits.global
 
-  private trait HasUpdateJob {
+  trait HasUpdateJob {
```

Comment:
Why not private ?

Reply:
I actually don't know, but when I ran it locally, it complained that the mockito framework could not mock `private static` classes. I used the same command as in the GitHub actions script.

Reply:
So I reverted it and will test it in the pipeline